content/200-concepts/100-components/02-prisma-client/052-advanced-type-safety/99-prisma-validator.mdx

```diff
@@ -72,6 +72,22 @@ const user = await prisma.user.findUnique({
 })
 ```
 
+Alternatively, you can use the following syntax that uses a "selector" pattern using an existing instance of Prisma Client:
+
+```ts
+import { Prisma } from '@prisma/client'
+import prisma from './lib/prisma'
+
+const userEmail = Prisma.validator(
+  prisma,
+  'user',
+  'findUnique',
+  'select'
+)({
+  email: true,
+})
+```
+
 The big difference is that the `userEmail` object is now type-safe. If you hover your mouse over it TypeScript will tell you the object's key/value pair. If you use dot notation to access the object's properties you will only be able to access the `email` property of the object.
```

Comment:
I think we may need to re-phrase the above section(s) as this next paragraph seems to refer to the previous example instead of both examples as a combined unit. Or maybe I'm overthinking things.

Reply:
I think we can keep this here since the example provides an alternative way to create the validator. 🤔